QuasarHelios2015/src/org/team1540/quasarhelios/AutonomousModeBase.java

```diff
@@ -51,6 +51,11 @@ protected void collectTote() throws AutonomousModeOverException,
 		waitUntil(BooleanMixing
 				.invert((BooleanInputPoll) QuasarHelios.autoLoader));
 	}
+	
+	protected void setForklift(float value) throws AutonomousModeOverException, InterruptedException {
```

Comment:
It's probably a good plan to keep the code consistent in terms of naming the Forklift/Clamp - we've called it Clamp so far, so it's probably a good plan to continue that.